### PR TITLE
Enable Audit Logs for Tinkerbell

### PR DESCRIPTION
**Pull request by tatlat**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jiayiwang7 — APPROVED** [review]

File: pkg/providers/tinkerbell/template.go [hunk 1]
```diff
@@ -362,6 +362,11 @@ func buildTemplateMapCP(
 	etcdTemplateOverride string,
 	datacenterSpec v1alpha1.TinkerbellDatacenterConfigSpec,
 ) (map[string]interface{}, error) {
+	auditPolicy, err := common.GetAuditPolicy(clusterSpec.Cluster.Spec.KubernetesVersion)
```
Comment (jiayiwang7) [inline pkg/providers/tinkerbell/template.go:365]: then you can move `values["auditPolicy"] = auditPolicy` to https://github.com/aws/eks-anywhere/blob/b1d360e9b52c030136d2d961b1f1756c29165852/pkg/providers/tinkerbell/template.go#L386

Comment (tatlat) [reply pkg/providers/tinkerbell/template.go:365]: yes

**Pull request merged**